src/org/labkey/test/util/WikiHelper.java

```diff
@@ -280,7 +282,13 @@ public void switchWikiToVisualView()
         {
             if (_test.isElementPresent(Locator.css("#wiki-tab-visual.labkey-tab-inactive")))
             {
-                _test.click(Locator.css("#wiki-tab-visual > a"));
+                Locator tab = Locator.css("#wiki-tab-visual > a");
+                _test.waitForElementToBeVisible(tab);
+                _test.click(tab);
+
+                Locator yesButton = Locator.tagWithText("span","Yes");
+                _test.waitForElementToBeVisible(yesButton);
+                _test.waitAndClick(yesButton);
```

Comment:
I don't think you need to do a waitForElementToBeVisible if you use waitAndClick. waitAndClick should wait for the element to be clickable.

Reply:
it doesn't. at least not in this instance. It fails to find the element pretty consistently w/o it.